packages/shared/src/components/ShareNewCommentPopup.tsx

```diff
@@ -97,7 +97,7 @@ export default function ShareNewCommentPopup({
         <Button
           className="btn-primary"
           buttonSize="small"
-          onClick={copyLink}
+          onClick={() => copyLink()}
```

Comment:
Why can't we keep the old implementation?

Reply:
The `copyLink` function now accepts parameters, and the `onClick` natively passes an `event` object, so it is giving an incompatible error.